member/index.md

```diff
@@ -78,21 +74,36 @@ We'd like you to bear in mind that none of our **moderators** work for nor are a
 | !t-systemrestore        | Displays info about System Restore                       |
 | !bsod                   | Displays info about diagnosing BSODs                     |
 | !patience               | Reminds you to be patient when asking questions          |
-| !linux-shilling         | Reminds you that random Linux shilling for the sake of it isnt helpful |
-| !at                     | Reminds you that mentioning someone who isn't active is not appreciated |
-| !join-insiders          | Gives you the Insiders role                              |
-| !join-insiders-slow     | Gives you the Insiders (Slow) role                       |
+
+### Staying updated Commands
+| Command                 | Function                                                 |
+| ----------------------- | -------------------------------------------------------- |
+| !join-insider-dev          | Gives you the Windows Insiders (Dev) role             |
+| !join-insider-beta     | Gives you the Windows Insiders (Beta) role                |
+| !join-insider-rp:     | Gives you the Windows Insiders (Release Preview) role      |
 | !join-patch-tuesday     | Gives you the 💻 Patch Tuesday role                      |
 | !keep-me-updated        | Gives you all opt-in roles                               |
 | !dont-keep-me-updated   | Takes away from you all opt-in roles                     |
+| $nj-latestbeta              | Gets the latest Beta Channel Insider build           |
+| $nj-latestdev         | Gets the latest Dev Channel Insider build                  |
+| $nj-latestrp              | Gets the latest Release Preview Channel Insider build  |
+| $nj-latestserver        | Gets the latest Insider Server build                     |
+
+### Misc Commands
+| Command                 | Function                                                 |
+| ----------------------- | -------------------------------------------------------- |
+| !rules                  | Posts a link to the server rules                                        |
+| !rank                   | DMs you your current rank                                |
+| !levels                 | DMs you the leaderboard for the server                   |
+| !whois `<member> `      | Displays data about the mentioned member                 |
```

Comment:
Unnecessary space after <member>

Reply:
Please bear with me as I try to update this on my end (still getting back into github after a good break)